code/weapon/weapons.cpp

```diff
@@ -5579,7 +5651,7 @@ int weapon_create( vec3d * pos, matrix * porient, int weapon_type, int parent_ob
 			// to reserve N signatures for the spawned weapons
 			if ( wip->wi_flags[Weapon::Info_Flags::Spawn] ){
                 multi_set_network_signature( (ushort)(Objects[objnum].net_signature + wip->total_children_spawned), MULTI_SIG_NON_PERMANENT );
```

Comment:
On weapon creation it 'reserves' net signatures for its children.

Reply:
Yes, this would be the problem.  You would have to find a way for it to calculate how many children it is going to spawn.

Reply:
It's variable, there is no guarantee for any specific number, and that's fundamental to this mechanic.